Cargo.toml

```
@@ -150,7 +150,7 @@ serde_json = { version = "1.0.128" }
 sha2 = { version = "0.10.8" }
 smallvec = { version = "1.13.2" }
 spdx = { version = "0.10.6" }
-syn = { version = "2.0.77" }
+syn = { version = "2.0.77", features = ["full"] }
```

```suggestion
syn = { version = "2.0.77" }
```

I don't think `features = ["full"]` is needed here?

---

Good catch, thanks!